src/ParcelRegistry.Consumer.Address/AddressConsumerItem.cs

```diff
@@ -109,6 +109,7 @@ public void Configure(EntityTypeBuilder<AddressConsumerItem> builder)
                     status => AddressStatus.Parse(status));
 
             builder.HasIndex(x => x.AddressId);
+            builder.HasIndex(x => x.Position);
```

Comment:
This is not a spatial index, if i'm not mistaken, a spatial index is already created in a previous PR.